platform/core/PolyPodCoreAndroid/core/src/main/kotlin/coop/polypoly/core/FeatureCategory.kt

```diff
@@ -0,0 +1,35 @@
+package coop.polypoly.core
+
+enum class FeatureCategoryId {
+    yourData,
+    knowHow,
+    tools,
+    developer
+}
+
+data class FeatureCategory(
+    val id: FeatureCategoryId,
```

Comment:
Please consider `type` or `kind` instead of ID.

Reply:
ID is used, as the same definition is in categories.json. I would argue that ID makes uniqueness explicit - no two categories with the same id can be in array.

Reply:
Well, types are also unique: stuff can only have one type (or kind)

Reply:
Kinda, for example in HomeScreen we have sections which have different ids, but same layout type. I will stick with id for now.